### DISCUSSION TITLE
Selector Switch

### DISCUSSION THREAD

**Discussion by ryanmurray** [nebhead/PiFire#67] [Q&A] [answered]
Good Day all,

I've begun to build a pi-fire based on the v2 board. The only thing im curious about is the selector switch. I plan to ditch my factory dead controller. Does the selector switch need to be installed or is the schematic default to pi-fire?

**Comment by nebhead** [comment, marked as answer]
No need to install a selector switch.  You can leave that un-populated.  Do keep in mind that you will need to have either a jumper or power switch on the PWRSW pins/header. 

![image](https://user-images.githubusercontent.com/19876886/148704942-d51fbd24-3f7c-4cf0-a9e2-c461afaee594.png)

I personally am just jumper-ing mine, but I used to have a power-switch to cycle power if needed.

**Comment by ryanmurray** [comment]
i was going to use a power switch on outside of the box just for ease of reset. This is a really cool project. My traegar controller died and has been sitting broke for around 6 months. Been using my camping little traegar which sucks for smoking fish or any sort of temperature control. Might fit this setup into one of those next.

**Reply by nebhead** [reply]
> i was going to use a power switch on outside of the box just for ease of reset. This is a really cool project. My traegar controller died and has been sitting broke for around 6 months. Been using my camping little traegar which sucks for smoking fish or any sort of temperature control. Might fit this setup into one of those next.

Very cool!  Post some pictures once you have it all up and running. :)
